apps/finance/app/src/components/TransfersFilters.js

```diff
@@ -16,152 +13,46 @@ const TransfersFilters = ({
   transferTypes,
   transferTypeFilter,
   onTransferTypeChange,
-  onDownload,
-}) => (
-  <Spring
-    native
-    config={springs.smooth}
-    from={{ progress: 0, height: 0 }}
-    to={{
-      progress: Number(opened),
-      height: opened || !compactMode ? 'auto' : 0,
-    }}
-    immediate={!compactMode}
-  >
-    {({ progress, height }) => (
-      <animated.div
-        style={{
-          width: '100%',
-          overflow: progress.interpolate(v => (v === 1 ? 'unset' : 'hidden')),
-          height,
-        }}
+}) => {
+  return (
+    <div
+      css={`
+        margin-bottom: ${1 * GU}px;
+        display: inline-grid;
+        grid-gap: ${1.5 * GU}px;
+        grid-template-columns: auto auto auto;
+      `}
+    >
+      <DropDown
+        label="Type"
```

Comment:
cc @dizzypaty Does it make sense for this to be renamed "Transfer type" or "Transaction type"?